src/WebJobs.Script.WebHost/App_Data/secrets/host.json

```diff
@@ -1,4 +1,4 @@
 ﻿{
-  "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy",
-  "functionKey": "zlnu496ve212kk1p84ncrtdvmtpembduqp25ajjc"
+    "masterKey": "t8laajal0a1ajkgzoqlfv5gxr4ebhqozebw4qzdy,z3dlq50s00cb3q3k11nil7xyt29ebst2n8rtn0t3",
```

Comment:
Yes, the primary scenario we're trying to address here is the ability to roll keys. It's usually going to be a single value. As David says, it might grow to 2 if someone needs to roll a key, but once their known consumers have moved to the new key, the old key will be removed.

Fabio is discussing a new scenario (feature creep) that this can also be used for - the ability to have N different consumer keys that you hand out to people. Not really a target scenario here, but the array can be used for that still, the values just won't have names.